datafusion/CHANGELOG.md

```diff
@@ -19,6 +19,250 @@
 
 # Changelog
 
+## [11.0.0](https://github.com/apache/arrow-datafusion/tree/11.0.0) (2022-08-16)
+
+[Full Changelog](https://github.com/apache/arrow-datafusion/compare/10.0.0-rc1...11.0.0)
+
+**Breaking changes:**
```

Comment:
this list seems a little light 😆  but I don't think we should spend too much time cleaning this up